js/src/alert.js

```diff
@@ -107,20 +106,6 @@ const Alert = (($) => {
       return closeEvent
     }
 
-    _removeElement(element) {
-      $(element).removeClass(ClassName.SHOW)
-
-      if (!Util.supportsTransitionEnd() ||
-          !$(element).hasClass(ClassName.FADE)) {
-        this._destroyElement(element)
-        return
-      }
-
-      $(element)
-        .one(Util.TRANSITION_END, (event) => this._destroyElement(element, event))
-        .emulateTransitionEnd(TRANSITION_DURATION)
-    }
-
```

Comment:
@Johann-S With the simplification due to`transition`, `_removeElement(element)` became a one liner and as it's called only once it seemed better to remove it and the one liner directly